corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -67,6 +68,21 @@ def convert_to_date(date):
     return result
 
 
+def get_days_since_last_form_submission_for_user_id(domain, user_id):
+    query = (
+        FormES()
+        .domain(domain)
+        .user_id(user_id)
+        .sort('received_on', desc=True)
+        .size(1)
```

Comment:
This could cause an exception if elastic is down. Should there be error handling here, or will that be handled at somewhere else on the call stack? I didn't see any tests for that type of error condition.

Reply:
this is a great point and would make downstream code potentially very fragile since this only gets run once a month. 

what kind of errors does elasticsearch typically throw in such a query if it's down?

Reply:
I guess `ESError`, yes? I will handle that in the downstream task

Reply:
I guess there is also `ConnectionTimeout`